.editorconfig

```diff
@@ -8,14 +8,12 @@ end_of_line = lf
 indent_style = space
 indent_size = 2
 insert_final_newline = true
-trim_trailing_whitespace = true
```

Comment:
Why?

Reply:
was a tryout for this config, buts to hard.
it fails e.g on this scenario if there are whitespaces between methods (eclipse default):
``` java
public class Test{
  public void method()
  {
  }
  <- whitespaces
  public void method2()
  {
  }
}
```